src/utils/scenarioParameters/VarTypesComponentsMapping.js

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import { GENERIC_VAR_TYPES_COMPONENTS_MAPPING } from './generic/VarTypesComponentsMapping';
+import { CUSTOM_VAR_TYPES_COMPONENTS_MAPPING } from './custom/ComponentsMapping';
```

Comment:
Oops, looks like we no longer have consistent names for files in _generic_ and _custom_ folders :sweat_smile: 
We could keep the name `VarTypesComponentsMapping` for both files, to use the same name as the file in the parent folder

Reply:
Yes, I saw it, it's coming in the next commit :laughing: